vendor/nke-watteco/th-sensor.yaml

```diff
@@ -0,0 +1,148 @@
+name: TH
+description: Temperature and Humidity sensor
+
+# Hardware versions (optional, use when you have revisions)
+hardwareVersions:
+  - version: '50-70-053'
+    numeric: 53
+  - version: '50-70-162'
+    numeric: 162
+
+# Firmware versions (at least one is mandatory)
+firmwareVersions:
+  - # Firmware version
+    version: '3.5.2'
+    numeric: 352
+    # Corresponding hardware versions (optional)
+    hardwareVersions:
+      - '50-70-053'
+      - '50-70-162'
+
+    # # Firmware features (optional)
+    # # Valid values are: remote rejoin (trigger a join from the application layer), transmission interval (configure how
+    # # often he device sends a message).
+    # features:
+    #   - remote rejoin
+    #   - transmission interval
+
+    # LoRaWAN Device Profiles per region
+    # Supported regions are EU863-870, US902-928, AU915-928, AS923, CN779-787, EU433, CN470-510, KR920-923, IN865-867,
+    # RU864-870
+    profiles:
+      EU863-870:
+        # Optional identifier of the vendor of the profile. When you specify the vendorID, the profile is loaded from
+        # the vendorID's folder. This allows you to reuse profiles from module or LoRaWAN end device stack vendors.
+        # If vendorID is empty, the current vendor ID is used. In this example, the vendorID is the current vendor ID,
+        # which is verbose.
+        vendorID: nke-watteco
```

Comment:
@Ben199-cpu please delete vendor ID from here

Reply:
Ok, do I have to delete it from each sensors' file ? Should I do another push of these files afterwards ?